src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx

```diff
@@ -139,7 +139,7 @@
     <value>Add an additional NuGet package source to use during installation.</value>
   </data>
   <data name="CommandDescription" xml:space="preserve">
-    <value>Install a tool for use on the command line.</value>
+    <value>Install global tool or add tool to local manifest and make it available.</value>
```

Comment:
@KathleenDollard to review. The end could be improved, I believe.

Reply:
Install global or local tool. Local tools are added to manifest and restored.

Reply:
I used the string above. I worked with Kathleen to come up with it